src/iop/colorbalancergb.c

```diff
@@ -1328,7 +1328,7 @@ void commit_params(struct dt_iop_module_t *self, dt_iop_params_t *p1, dt_dev_pix
 
 void init_pipe(dt_iop_module_t *self, dt_dev_pixelpipe_t *pipe, dt_dev_pixelpipe_iop_t *piece)
 {
-  piece->data = calloc(1, sizeof(dt_iop_colorbalancergb_data_t));
+  piece->data = dt_calloc_align(64, sizeof(dt_iop_colorbalancergb_data_t));
```

Comment:
Maybe a corresponding `dt_free_align()` in `cleanup_pipe()`?

Reply:
Yes, you're right.